src/System.Net.WebSockets.WebSocketProtocol/src/System.Net.WebSockets.WebSocketProtocol.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net48</TargetFrameworks>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>6.0.0</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">6.1.0</VersionPrefix>
+  </PropertyGroup>
```

Comment:
PackageValidationBaselineVersion?

Reply:
The nupkg has a netcoreapp2.1 folder, and it contains a DLL with an implementation (it's not `_._`). But since netcoreapp2.1 is out of support, should I add that to the tests project?

It seems that when we target net6.0 and try to consume an API from this assembly, we do use the one from the nupkg: `bin/Debug/net6.0/System.Net.WebSockets.WebSocketProtocol.dll` as described in the PR description.

In other words: We did not make this assembly in-box, so we need to test it in net6.0, yes.

https://github.com/dotnet/runtime/tree/release/6.0/src/libraries

Reply:
> What about AssemblyVersion?

This package is behaving as completely out of band, so it can use the repo-wide versioning similar to BCL.HashCode.  We only need to set the AssemblyVersion when an assembly has overlap with a framework assembly.

> In other words: We did not make this assembly in-box, so we need to test it in net6.0, yes.

Correct.  Your test can use whatever .NETCore version we're using for other tests.

Reply:
Speaking of which -- dotnet pack is failing because it can't find the version I specified. The error message says:

```
C:\Users\calope\source\repos\maintenance-packages\src\System.Net.WebSockets.WebSocketProtocol\src\System.Net.WebSockets.WebSocketProtocol.csproj : error NU1102: Unable to find package System.Net.Web
Sockets.WebSocketProtocol with version (= 6.0.0)
C:\Users\calope\source\repos\maintenance-packages\src\System.Net.WebSockets.WebSocketProtocol\src\System.Net.WebSockets.WebSocketProtocol.csproj : error NU1102:   - Found 37 version(s) in dotnet-pub
lic [ Nearest version: 6.0.0-preview.1.21102.12 ]
C:\Users\calope\source\repos\maintenance-packages\src\System.Net.WebSockets.WebSocketProtocol\src\System.Net.WebSockets.WebSocketProtocol.csproj : error NU1102:   - Found 1 version(s) in dotnet-eng
[ Nearest version: 5.0.0-alpha.1.19618.1 ]
C:\Users\calope\source\repos\maintenance-packages\src\System.Net.WebSockets.WebSocketProtocol\src\System.Net.WebSockets.WebSocketProtocol.csproj : error NU1102:   - Found 0 version(s) in dotnet9
```

Based on the errors, it sounds to me that I need to set these properties:

```xml
    <VersionPrefix>5.0.0</VersionPrefix>
    <VersionPrefix Condition="'$(IsPackable)' == 'true'">6.0.0</VersionPrefix>
    <PackageValidationBaselineVersion>5.0.0</PackageValidationBaselineVersion>
```

If I run dotnet pack again, it passes:

```
❯ dotnet pack .\src\System.Net.WebSockets.WebSocketProtocol\src\System.Net.WebSockets.WebSocketProtocol.csproj
MSBuild version 17.9.4+90725d08d for .NET
  Determining projects to restore...
  Restored C:\Users\calope\source\repos\maintenance-packages\src\System.Buffers\src\System.Buffers.csproj (in 353 ms).
  Restored C:\Users\calope\source\repos\maintenance-packages\src\System.Memory\src\System.Memory.csproj (in 353 ms).
  Restored C:\Users\calope\source\repos\maintenance-packages\src\System.Net.WebSockets.WebSocketProtocol\src\System.Net.WebSockets.WebSocketProtocol.csproj (in 353 ms).
  Restored C:\Users\calope\source\repos\maintenance-packages\src\System.Threading.Tasks.Extensions\src\System.Threading.Tasks.Extensions.csproj (in 353 ms).
  System.Net.WebSockets.WebSocketProtocol -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Net.WebSockets.WebSocketProtocol\Release\net6.0\System.Net.WebSockets.WebSocketPro
  tocol.dll
  System.Threading.Tasks.Extensions -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Threading.Tasks.Extensions\Release\netstandard2.0\System.Threading.Tasks.Extensions.dll
  System.Buffers -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Buffers\Release\netstandard2.0\System.Buffers.dll
  System.Threading.Tasks.Extensions -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Threading.Tasks.Extensions\Release\net462\System.Threading.Tasks.Extensions.dll
  System.Buffers -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Buffers\Release\net462\System.Buffers.dll
  System.Memory -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Memory\Release\netstandard2.0\System.Memory.dll
  System.Net.WebSockets.WebSocketProtocol -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Net.WebSockets.WebSocketProtocol\Release\netstandard2.0\System.Net.WebSockets.WebS
  ocketProtocol.dll
  System.Memory -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Memory\Release\net462\System.Memory.dll
  System.Net.WebSockets.WebSocketProtocol -> C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Net.WebSockets.WebSocketProtocol\Release\net462\System.Net.WebSockets.WebSocketPro
  tocol.dll
  The package System.Net.WebSockets.WebSocketProtocol.6.0.0-dev.24129.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Net.WebSockets.WebSocketProtocol.6.0.0-dev.24129.1.nupkg'.
```

Reply:
Fixed by setting PackageValidationBaselineVersion and correcting VersionPrefix (both).